src/external/VTK.jl

```diff
@@ -0,0 +1,49 @@
+# Uses WriteVTK.jl to convert scfres structure to VTk file format
+import WriteVTK: vtk_grid, vtk_save
+
+"""
+    save_scfres(VTKFileName, scfres)
+
+The function takes in the VTK filename and the scfres structure and stores into a VTK file.
+Grid Values:
+ρ_real -> real values of scfers.ρ
+ψreal_i_j -> Real values of Bloch waves in real space, where i and j are Kpoint and EigenVector indexes respectively
+ρspin -> Real of ρspin are stored if ρspin in present
+MetaData:
+Energy and occupation.
+"""
+function save_scfres(VTKFileName::AbstractString, scfres)
+    # Initialzing the VTK Grid
+    basis = scfres.basis
+    basis_r_co = r_vectors_cart(basis)
+    gridsize = size(basis_r_co)
+    grid = zeros(3, gridsize...)
+    for (idcs, r) in zip(CartesianIndices(gridsize), basis_r_co)
+        grid[:, Tuple(idcs)...] = r
+    end
+    vtkfile = vtk_grid(VTKFileName, grid)
+    
+    # Storing the Bloch Waves in Real space
+    for i in 1:length(basis.kpoints)
+        for j in 1:7
```

Comment:
Why `7`? Don't rely on there being `7` bands :smile:.

Reply:
I forgot to add stashed changes . My bad. 😞 . I have update the function in the recent commit.